### ISSUE TITLE
CEX in Python 2 [SSL: TLSV1_ALERT_PROTOCOL_VERSION]

### ISSUE THREAD

**Issue by mmehrle** [ccxt/ccxt#1139]
- OS: OS.X 10.11.4
- Programming Language: python
- CCXT version: 1.10.503
- Exchange: cex
- Method: exchange.load_markets()

```
import sys
import ccxt

def main(argv):
    global EXCHANGE

    if (len(argv) > 1):
        EXCHANGE = argv[1:]
    exchange = getattr(ccxt, EXCHANGE)({
        # 'verbose': True,
    })

    print("market id: {}".format(exchange.id))
    markets = exchange.load_markets()

if __name__ == '__main__':
    sys.exit(main(sys.argv))
```

Error: ccxt.base.errors.ExchangeNotAvailable: cex GET https://cex.io/api/currency_limits/ HTTPSConnectionPool(host='cex.io', port=443): Max retries exceeded with url: /api/currency_limits/ (Caused by SSLError(SSLError(1, u'[SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590)'),))

**Comment by kroitor** [comment]
This is fixed in most current version of CCXT, please update and retry once again. Feel free to reopen it or just ask questions if you still have difficulties with it after that. Thx!

**Comment by mmehrle** [comment]
Sorry about that! Thanks again for the almost instant response 👍

**Comment by kroitor** [comment]
@mmehrle we will be happy if you report back whether it works normally for you or not. Thx!

**Comment by mmehrle** [comment]
Hey guys - sorry for being the bearer of bad news but I'm still getting the same error:
`
CCXT version: 1.10.614
market id: cex
Traceback (most recent call last):
  File "/Users/xxx/crypto_dev/ccxt_query_exchange.py", line 100, in <module>
    sys.exit(main(sys.argv))
  File "/Users/xxx/crypto_dev/ccxt_query_exchange.py", line 24, in main
    print_market_overview(argv)
  File "/Users/xxx/crypto_dev/ccxt_query_exchange.py", line 35, in print_market_overview
    markets = exchange.load_markets()
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/base/exchange.py", line 829, in load_markets
    markets = self.fetch_markets()
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/cex.py", line 83, in fetch_markets
    markets = self.publicGetCurrencyLimits()
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/cex.py", line 390, in request
    response = self.fetch2(path, api, method, params, headers, body)
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/base/exchange.py", line 296, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/base/exchange.py", line 355, in fetch
    self.raise_error(ExchangeNotAvailable, url, method, e)
  File "/Users/michael/quant-2.7/lib/python2.7/site-packages/ccxt/base/exchange.py", line 278, in raise_error
    details,
ccxt.base.errors.ExchangeNotAvailable: cex GET https://cex.io/api/currency_limits/ HTTPSConnectionPool(host='cex.io', port=443): Max retries exceeded with url: /api/currency_limits/ (Caused by SSLError(SSLError(1, u'[SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590)'),)) 
`

**Comment by kroitor** [comment]
Ok, we will look into it and will get back to you with an update on this. Thx!

**Comment by kroitor** [comment]
Can you report your version of OpenSSL ? `python -c "import ssl; print(ssl.OPENSSL_VERSION)"` ?

**Comment by mmehrle** [comment]
Hey kroitor - good idea but I seem to be sufficiently secure shelled ;-)

OpenSSL 1.0.2k  26 Jan 2017

**Comment by mmehrle** [comment]
Actually that was it - I added it to my script which was run out of PyCharm and got this:

CCXT version: 1.10.614
SSL versionOpenSSL 0.9.8zh 14 Jan 2016

For some strange reason I'm unable to upgrade to a higher version. Right out of the virtual env directory I'm using for my PyCharm project:

/Users/xxx/quant-2.7/bin
MBP:bin xxx$ ./python -c "import ssl; print(ssl.OPENSSL_VERSION)"
OpenSSL 0.9.8zh 14 Jan 2016

Does anyone know how to pull a compatible version? I tried a myriad of commands but all pip offers is 1.5 or 1.6 and those are incompatible.

Sorry, I know this is unrelated to ccxt - so feel free to ignore.
